ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py

```diff
@@ -176,3 +178,48 @@ def test_tensorflow_slogdet(
         fn_tree="linalg.slogdet",
         x=np.asarray(x, dtype=input_dtype),
     )
+
+
+# tensordot
+@handle_cmd_line_args
+@given(
+    dtype_x1_x2_axis=_get_dtype_value1_value2_axis_for_tensordot(
+        available_dtypes=helpers.get_dtypes("numeric"),
+    ),
+    as_variable=st.booleans(),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.tensorflow.tensordot"
+    ),
+    native_array=st.booleans(),
+)
+def test_tensorflow_tensordot(
+    *,
+    dtype_x1_x2_axis,
+    as_variable,
+    num_positional_args,
+    native_array,
+    fw,
+):
+    (
+        dtype,
+        x1,
+        x2,
+        axis,
+    ) = dtype_x1_x2_axis
+
+    as_variable = [as_variable, as_variable]
+    native_array = [native_array, native_array]
+
+    helpers.test_frontend_function(
+        input_dtypes=dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
+        frontend="tensorflow",
+        fn_tree="tensordot",
+        x=x1,
+        x2=x2,
```

Comment:
Is this supposed to be `x2` or `y`? The frontend function uses `x` and `y` as its arguments.